packages/libs/ui/index.ts

```diff
@@ -16,31 +17,45 @@ export {
   StateCode,
 } from './SceneryTownSource';
 
+const baseMapStyle: StyleSpecification = {
+  version: 8,
+  // free font glyphs, required when adding text-fields.
+  // https://github.com/openmaptiles/fonts
+  glyphs: 'https://fonts.openmaptiles.org/{fontstack}/{range}.pbf',
+  // sources and layers are empty because they're declared as child
+  // components below.
```

Comment:
A very outdated comment.

```suggestion
  // sources and layers are empty because they're declared as children
  // of consuming `MapGl` components.
```